_layouts/home.html

```diff
@@ -31,5 +31,15 @@ <h1 class="page-heading center">Contributors</h1>
         <i class="fa fa-facebook"></i>
       </a>
     </li>
+    
+    <li>
+     <a href="https://github.com/ashunimbz">
+	 Ashish Nimbalkar
+     </a>
+     <a href = "https://facebook/ashunimbz">
+          <i class="fa fa-facebook"></i>
```

Comment:
Hey @ashunimbz - Everything seems good, but could you remove empty lines at top and bottom (line 34, 43) and fix the indentation.